src/styles/utils.ts

```diff
@@ -51,3 +52,21 @@ export const flexCenter = (
 export const absoluteFill = () => ({
   ...StyleSheet.absoluteFillObject,
 });
+
+export function tokenName(value: string | number) {
```

Comment:
This function seems redundant is basically a no-op.

Reply:
Doing `tokens[key] = value;` should be enough, right?

Reply:
Yes, the tokenName was originally something different (based on Full-Stack Template) and it resulted being just value after a few iterations. Forgot to remove it. Good catch!